playbooks/appsemblerPlaybooks/ficus_basic.yml

```diff
@@ -3,6 +3,7 @@
 # Appsembler basic tier deployment
 
 - include: monitoring.yml
+  when: SKIP_MONITORING is undefined
```

Comment:
@tkeemon Good point on that, thanks. What I want is to avoid having to locally modify the playbook to skip installing monitoring for staging/dev servers

Main driver for this for me is that I ran into install failures for `filebeat` 

So I guess I've got the same question that TJ does on disabling monitoring for staging servers (noting that some staging servers are set up as more dev than customer "sanity-check" staging servers)